lib/rubocop/cop/layout/space_inside_hash_literal_braces.rb

```diff
@@ -85,12 +85,13 @@ def on_hash(node)
         private
 
         def hash_literal_with_braces(node)
-          tokens = processed_source.tokens
-          begin_index = index_of_first_token(node)
-          return unless tokens[begin_index].left_brace?
+          begin_index = processed_source.send(:first_token_index, node)
```

Comment:
We should have made this methods public also, I think.

Reply:
Maybe, but I am not sure this is a good example here. Why not call `tokens_within`, use those and yield those?

Reply:
Agreed, updated.
Do you have an idea on why CI fails on windows only?

Reply:
Not sure and I've got to run 🏃‍♂️

Reply:
Released 0.4.2; if you bump the requirement this should be good to go 👍